### PR TITLE
Feature/bundle issues

### PR DESCRIPTION
**Pull request by fraincs**
Issue: 

## Summary

#805 / #804 / #810 / #811 

## What I did

Fixed above issues

### CODE REVIEW

**Review by alexasselin008 — APPROVED** [review]

File: packages/components/src/input/src/Input.css [hunk 1]
```diff
@@ -34,11 +34,20 @@
...
     background-color: var(--o-ui-bg-alias-input-selection);
 }
 
+/* AUTOFILL */
+input:-webkit-autofill,
+textarea:-webkit-autofill,
+select:-webkit-autofill {
+    -webkit-box-shadow: 0 0 0 1000px var(--o-ui-bg-alias-accent-faint) inset !important;
+    -webkit-text-fill-color: var(--o-ui-text-alias-primary) !important;
+}
```
Comment (patricklafrance) [inline packages/components/src/input/src/Input.css:49]: Your selectors are too generic, you should only match orbit elements meaning, `.o-ui-input` selectors.

Comment (fraincs) [reply packages/components/src/input/src/Input.css:49]: Totally, copy pasta gone wrong

Comment (patricklafrance) [reply packages/components/src/input/src/Input.css:48]: Also, it's not a big deal but, adding code for "select" in this file is not great from a maintenance standpoint. This file is scope to text inputs.

**Comment by fraincs** [comment]
Yup it has been removed.

**Pull request merged**